process/process_darwin.go

```diff
@@ -389,12 +389,31 @@ func convertCPUTimes(s string) (ret float64, err error) {
 	var _tmp string
 	if strings.Contains(s, ":") {
 		_t := strings.Split(s, ":")
-		hour, err := strconv.Atoi(_t[0])
-		if err != nil {
+		if len(_t) > 3 {
```

Comment:
@keefo In this method, multiple nested `if` clause is used and it leads to low readability. What about using `switch`  for better readability?

Reply:
@mznet  No problem, I will do that.